doc/tutorials/polymers/polymers.ipynb

```diff
@@ -4,7 +4,78 @@
    "cell_type": "markdown",
```

Comment:
<p>"prove this formula computationally" I would avoid the word 'proof' here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/4052/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>